index.js

```diff
@@ -23,7 +23,7 @@ const regexpTree = {
    * @return Object AST
    */
   parse(regexp) {
-    return parser.parse(regexp);
+    return parser.parse(`${regexp}`);
```

Comment:
I avoid calling `toString` in JavaScript directly because it is just one step in `ToString` algorithm, along with `@@toPrimitive` and `valueOf`.

If I pass an object with valid `@@toPrimitive` or `valueOf` methods I expect them to be called. Also, `.toString()` on symbols will return description string, rather then throw like `ToString`. `RegExp` constructor uses `ToString`.

What about

```js
const ToString = v => `${v}`

// ...

return parser.parse(ToString(v))
```